src/vulkan-renderer/vk_tools/gpu_info.cpp

```diff
@@ -145,6 +147,7 @@ void print_device_layers(const VkPhysicalDevice gpu) {
     spdlog::debug("Number of device layers: {}", device_layer_count);
 
     if (device_layer_count == 0) {
+        // This is not an error. Some platforms simply don't have any device layers.
```

Comment:
Maybe remove the device layer stuff completely since they are deprecated?

Reply:
I know they are deprecated but I am undecided about removing them. I will read up about it.

Reply:
Well, no device we support uses them since they were deprecated in vulkan 1.0

Reply:
The Vulkan spec states:

> Device-only layers are now deprecated, and this specification no longer distinguishes between instance and device layers. Layers are enabled during instance creation, and are able to intercept all commands operating on that instance or any of its child objects. **At the time of deprecation there were no known device-only layers and no compelling reason to create one.**

from:
https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#extendingvulkan-layers-devicelayerdeprecation

So yeah I will remote it :)

Reply:
Done.